src/utils/platform-colors.js

```diff
@@ -20,6 +20,11 @@ export const MOVING_PLATFORMS_COLORS = [
   "#a6cee3",
   "#1f78b4",
   "#fb9a99",
+  "#673AB7",
+  "#F2B705",
+  "#69544F",
+  "#AB47BC",
+  "#B9A7A2",
```

Comment:
```suggestion
  "#fb9a99",
  "#e31a1c",
  "#cab2d6",
  "#6a3d9a",
  "#ffff99",
  "#b15928",
```
@willemarcel the original 8 colors come from the Color Brewer 8 color palette. I would recommend that we stick with this mechanism. I see that we can activate a 12 color pairing: https://colorbrewer2.org/#type=qualitative&scheme=Paired&n=12. I've inserted these color pairs into the original list.